admin/model/brand.php

```diff
@@ -0,0 +1,71 @@
+<?php
+include('database.php');
+// Returns all brands in the BRAND table
+// as a multi-dimensional associative array
+function getBrands() {
+    include('database.php');
+
+    $query = 'SELECT * FROM BRAND';
+   $result = mysqli_query($connection, $query);
+
+    if($result) {
+       return mysqli_fetch_all($result, MYSQLI_ASSOC);
+    } else {
+        return "Error: getBrands()";
+    }
+}
+
+?>
+<?php
+/*
+
+*/
+/*
+*   AFTER SUBMITTING THE UPDATE FORM, UPDATE THE INFO
+*   IN THE DATABASE
+*/
+
+function updateDB(){
```

Comment:
I would make this function a little more specific to brands. So something like updateBrand()